src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -97,61 +97,63 @@ private AdsDataProvider() {
   public static Object[][] skinWithTheme() {
     return new Object[][]{
         {
-            WIKI_REGULAR, "Skin",
-            new Dimension(1200, 1000),
-            NO_SKIN_LEFT, NO_SKIN_RIGHT,
-            null,
-            null
-        }, {
+//            WIKI_REGULAR, "Skin",
+//            new Dimension(1200, 1000),
+//            NO_SKIN_LEFT, NO_SKIN_RIGHT,
+//            null,
+//            null
+//        }, {
             WIKI_REGULAR, "Skin",
             new Dimension(1600, 900),
             SKIN_LEFT, SKIN_RIGHT,
             "AAAAAA",
             "FFFFFF"
-        }, {
-            WIKI_REGULAR, "Skin",
-            WindowSize.DESKTOP,
-            SKIN_LEFT, SKIN_RIGHT,
-            "AAAAAA",
-            "FFFFFF"
-        }, {
-            WIKI_REGULAR, "Skin",
-            new Dimension(2400, 1080),
-            SKIN_LEFT, SKIN_RIGHT,
-            "AAAAAA",
-            "FFFFFF"
-        }, {
-            WIKI_REGULAR, "Skin/NoMiddleColor",
-            WindowSize.DESKTOP,
-            SKIN_LEFT, SKIN_RIGHT,
-            "AAAAAA",
-            ""
         }
+//        {
+//            WIKI_REGULAR, "Skin",
+//            WindowSize.DESKTOP,
+//            SKIN_LEFT, SKIN_RIGHT,
+//            "AAAAAA",
+//            "FFFFFF"
+//        }, {
+//            WIKI_REGULAR, "Skin",
+//            new Dimension(2400, 1080),
+//            SKIN_LEFT, SKIN_RIGHT,
+//            "AAAAAA",
+//            "FFFFFF"
+//        }, {
+//            WIKI_REGULAR, "Skin/NoMiddleColor",
+//            WindowSize.DESKTOP,
+//            SKIN_LEFT, SKIN_RIGHT,
+//            "AAAAAA",
+//            ""
+//        }
     };
   }
 
   @DataProvider
   public static Object[][] skinWithoutTheme() {
     return new Object[][]{
+//        {
+//            WIKI_SPECIAL, "SyntheticTests/Skin",
+//            new Dimension(1600, 900),
+//            SKIN_LEFT, SKIN_RIGHT,
+//            "AAAAAA",
+//            "FFFFFF"
+//        }, {
+//            WIKI_SPECIAL, "SyntheticTests/Skin",
+//            WindowSize.DESKTOP,
+//            SKIN_LEFT, SKIN_RIGHT,
+//            "AAAAAA",
+//            "FFFFFF"
+//        }, {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)